### ISSUE TITLE
Broken link to fontawesome/brands icons

### ISSUE THREAD

**Issue by radek-chramosil** [jbms/sphinx-immaterial#337]
Hi there,

I thought I would let you know that you often pointing to fontawesome brands icons in the documentation. However, when I click on the link, it takes me to the repo, however the icons are not there.]

This is the broken link.

https://github.com/squidfunk/mkdocs-material/tree/master/material/.icons

I am not sure where to look up the right set of icons. [mkdocs-material](https://github.com/squidfunk/mkdocs-material) moved the icons.

Best,
Radek

**Comment by 2bndy5** [comment]
Yeah, the mkdocs-material theme has undergone a refactor since that doc was written. They now exist in https://github.com/squidfunk/mkdocs-material/tree/master/material/templates/.icons

IMO, that link (even when it worked) is not very helpful because github only displays a portion of the files in a large folder. I'd prefer using the [links listed in the inline-icons doc](https://jbms.github.io/sphinx-immaterial/inline_icons.html#using-inline-icons), but that doesn't help describe the folder structure used for the vendors of the icons. Some examples:
- [`material/alert`](https://github.com/squidfunk/mkdocs-material/blob/master/material/templates/.icons/material/alert.svg) for the [alert icon vendored by the MDI npm package](https://pictogrammers.com/library/mdi/icon/alert/)
- [`fontawesome/regular/bell`](https://github.com/squidfunk/mkdocs-material/blob/master/material/templates/.icons/fontawesome/regular/bell.svg) for the [bell icon vendored by the fortawesome npm package](https://fontawesome.com/icons/bell?f=classic&s=regular)
